cmd/gcp-controller-manager/kubelet_readonly_csr_approver.go

```diff
@@ -327,3 +329,28 @@ func validateRbac(request kubeletReadonlyCSRRequest) kubeletReadonlyCSRResponse
 		message: fmt.Sprintf("user %s is allowed to create a CSR", csr.Spec.Username),
 	}
 }
+
+// validateCommonName will make sure the common name for x509 has kubelet-ro-client
+func validateCommonName(request kubeletReadonlyCSRRequest) kubeletReadonlyCSRResponse {
```

Comment:
where is this function being called other than in the tests?

Reply:
Will be in next commit